garage/algos/cem.py

```diff
@@ -7,7 +7,9 @@
 from garage.core import Serializable
 import garage.misc.logger as logger
 from garage.misc.special import discount_cumsum
-import garage.plotter as plotter
+from garage.plotter import Plotter
+from garage.sampler import parallel_sampler
```

Comment:
where are parallel_sampler and stateful_pool used?

Reply:
parallel_sampler is used in line 113 and 189, while stateful_pool is used in line 134. Theses imports were added since both parallel_sampler and stateful_pool were detected as uninitialized imports by pylint.

Reply:
lgtm